app/core/src/main/java/com/fsck/k9/mailstore/LocalFolder.java

```diff
@@ -313,6 +313,18 @@ public boolean create() throws MessagingException {
         return true;
     }
 
+    public static void createLocalFolder(LocalStore localStore, String internalId, String folderName)
+            throws MessagingException {
+        LocalFolder folder = localStore.getFolder(internalId);
+        if (!folder.exists()) {
+            folder.create();
+        }
+        folder.setName(folderName);
+        folder.setInTopGroup(true);
```

Comment:
The method name is misleading, because this doesn't create a "simple" local folder. Maybe `createTopGroupLocalFolder`? 
Or drop this line. The outbox folder is special cased in `FolderRepository` to always show below the inbox anyway.

Reply:
Dropped the line after moving this method to `LocalStore`